test/test-applications/integrations/dependency-libs/LogsInjectionHelper/LogsInjectionHelper.csproj

```diff
@@ -0,0 +1,12 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net45;net461;netstandard2.0</TargetFrameworks>
+    <LangVersion>latest</LangVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\..\..\src\Datadog.Trace\Datadog.Trace.csproj" />
```

Comment:
I thought there were issues with code coverage when referencing the Datadog.Trace.dll directly, but I see some other test-apps do it, so I'm not sure now. @kevingosse?

Reply:
@kevingosse we recently merged everything into one `Datadog.Trace.dll` and we have a bunch of applications that directly reference it. Is our code coverage okay or do we need to change our test applications somehow?